src/erlcloud_s3.erl

```diff
@@ -506,7 +511,7 @@ put_bucket_lifecycle(BucketName, Policy, Config)
     put_bucket_lifecycle(BucketName, list_to_binary(XmlPolicy), Config);
 put_bucket_lifecycle(BucketName, XmlPolicy, Config)
   when is_list(BucketName), is_binary(XmlPolicy), is_record(Config, aws_config) ->
-    Md5 = base64:encode(crypto:hash(md5, XmlPolicy)),
+    Md5 = base64:encode(erlang:md5(XmlPolicy)),
```

Comment:
Why the change from `crypto:hash` to `erlang:md5`? I distantly remember we had to go in the other direction once, years ago.